mex/editor/components.py

```diff
@@ -38,23 +38,29 @@ def render_link(value: EditorValue) -> rx.Component:
     )
 
 
+def render_span(text: str | None) -> rx.Component:
```

Comment:
Why use a span here? why not plain text?

Reply:
the default rx.Text will output a `<p>`, which is unsuitable for inline text snippets